validator/src/commands/set_public_address/mod.rs

```diff
@@ -1,11 +1,39 @@
 use {
-    crate::{admin_rpc_service, cli::DefaultArgs},
+    crate::{admin_rpc_service, cli::DefaultArgs, commands::FromClapArgMatches},
     clap::{App, Arg, ArgGroup, ArgMatches, SubCommand},
     std::{net::SocketAddr, path::Path, process::exit},
 };
 
+const COMMAND: &str = "set-public-address";
+
+#[derive(Debug, PartialEq)]
+pub struct SetPublicAddressArg {
```

Comment:
nit: `SetPublicAddressArg` ==> `SetPublicAddressArgs`

Reply:
[1053e09](https://github.com/anza-xyz/agave/pull/4922/commits/1053e09d962df9e70cfa0d4e94593890f4527c00) 🫡

Reply:
due to the rebase. now is [638631f](https://github.com/anza-xyz/agave/pull/4922/commits/638631fc6e4321df3bac3b279856e2d60b1e72d0)